Dynavity/Dynavity/graph/Graph.swift

```diff
@@ -248,9 +248,6 @@ struct Graph<T: Hashable> {
 
     /// Checks the representation invariants.
     private func checkRepresentation() -> Bool {
-        // TODO: remember to disable this / ship app in production
-        // The next line is here for ease of commenting / uncommenting this function out
-        true ||
```

Comment:
Since `checkRepresentation` is only ever called in assertions, we just need to build the app in release mode.